integration/tests/cook/test_basic.py

```diff
@@ -2465,8 +2465,44 @@ def test_pools_in_default_limit_response(self):
                 resp = util.get_limit(self.cook_url, limit, user)
                 self.assertFalse('pools' in resp.json())
 
+    @pytest.mark.serial
```

Comment:
We can skip this test if the `DATA_LOCAL_SERVICE` environment variable is not set.